### DISCUSSION TITLE
Implement method of including upper limits in the model fit

### DISCUSSION THREAD

**Discussion by cplee1** [NickSwainston/pulsar_spectra#61] [General]
Some papers include lower or upper limits on flux density measurements. This is an example of upper limits on a Jankowski 2018 plot. He plotted them but did not include them in the fit.
![censored_example](https://user-images.githubusercontent.com/96163140/168238640-bea72be4-462b-4a03-80ac-1f1818c7e877.png)

At some point, it may be useful to implement an option to include censored data in the fit using a statistical method such as the Tobit model. Alternatively, or perhaps as well, we could implement a feature to plot the data (with arrows indicating limits) whilst not including them in the fit, as seen in papers like Jankowski et al and Bilous et al.

Papers with censored data:
- Murphy (2017). ADS: https://ui.adsabs.harvard.edu/abs/2017PASA...34...20M/abstract
- Sieber (1973). ADS: https://ui.adsabs.harvard.edu/abs/1973A%26A....28..237S/abstract
- Bilous (2016). ADS: https://ui.adsabs.harvard.edu/abs/2016A%26A...591A.134B/abstract
- Bilous (2020). ADS: https://ui.adsabs.harvard.edu/abs/2020A%26A...635A..75B/abstract

Useful Links:
- https://en.wikipedia.org/wiki/Censoring_(statistics)#Censored_regression
- https://en.wikipedia.org/wiki/Tobit_model

**Comment by NickSwainston** [comment]
Having upper limits would be an excellent way to confirm spectral turnovers to confirm not just scintillation
